app/controllers/charges_controller.rb

```diff
@@ -20,8 +20,9 @@ def create
     is_human = verify_recaptcha(action: "donate", minimum_score: 0.5, secret_key: ENV["RECAPTCHA_V3_SECRET_KEY"])
     # if users is a bot then return error
     unless is_human
+      message = "Oops! Something went wrong. Please try again"
```

Comment:
Maybe will be useful to scope the message so we known what is all about. What do you think?
Something like: _"Unable to validate you are not a bot"_

In any case, I don't expect this to happen...